services/frontend-service/src/ui/components/ReleaseCard/ReleaseCard.scss

```diff
@@ -44,6 +44,7 @@ Copyright 2023 freiheit.com*/
 .release-card__header {
     .release__title {
         @extend .text-bold;
+        font-size: 14px;
```

Comment:
I think it's a slippery slope to be overwriting the typography classes.
There was a story in the past to re-align the design because we started adding text modifications in too many places and it became unmaintainable :(